ui/client/containers/Visualization.js

```diff
@@ -58,7 +58,12 @@ class Visualization extends React.Component {
 
   bindPasteShortcut() {
     return (event) => this.props.actions.pasteSelection(
-      () => this.pasteSelection(event),
+      () => {
+        if (["INPUT", "TEXTAREA"].includes(event.target?.tagName)) {
```

Comment:
does this mean if I copy nodes and "paste" them to textarea I'll get json?

Reply:
yes. if user has focus on input field then he wants to fill this input. pasting clipboard content somewhere else is confusing